__tests__/app.test.js

```diff
@@ -399,4 +399,25 @@ describe('PATCH /api/reviews/:review_id', () => {
             });
     });
 
-});
\ No newline at end of file
+});
+
+describe('Get /api/users', () => {
+    test("200: response with array of all the user objects", () => {
+        return request(app)
+            .get("/api/users")
+            .expect(200)
+            .then( res => {
+                const { users } = res.body;
+                expect(users).toBeInstanceOf(Array);
+                users.forEach((user) => {
```

Comment:
You MUST assert the length of the array here - otherwise the forEach will never run correctly